packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -223,10 +223,13 @@ class TextPainter {
   /// parameters, lays it out with the supplied [minWidth] and [maxWidth], and
   /// returns its [TextPainter.width] making sure to dispose the underlying
   /// resources.
```

Comment:
we should add something about how this is expensive

Reply:
Done